soc/infineon/cat1b/cyw20829/Kconfig.soc

```diff
@@ -26,6 +26,22 @@ config SOC_PACKAGE_CYW20829_40_QFN
 config SOC_PACKAGE_CYW20829_77_BGA
 	bool
 
+config CYW20829_FLASH_SAHB_ADDR
+	hex
+	default $(dt_nodelabel_reg_addr_hex,flash_sahb)
+
+config CYW20829_FLASH_CBUS_ADDR
+	hex
+	default $(dt_nodelabel_reg_addr_hex,flash_cbus)
+
+config CYW20829_SRAM_SAHB_ADDR
+	hex
+	default $(dt_nodelabel_reg_addr_hex,sram_sahb)
+
+config CYW20829_SRAM_CBUS_ADDR
+	hex
+	default $(dt_nodelabel_reg_addr_hex,sram_cbus)
```

Comment:
these need to move to `Kconfig` file as they have no use in sysbuild (and there is no devicetree even)